bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -108,11 +121,6 @@ public class CloudClient {
      */
     private boolean isConnected;
 
-    /*
-     * This variable holds version of local openHAB
-     */
-    private String openHABVersion;
```

Comment:
Was the openHABVersion always empty and thus cleaned up in this PR?

Reply:
I think a long time ago it was needed, but at some point the code was changed to use `OpenHAB.getVersion()` so this variable and getter/setter was not used, so dead code.